main/common/resources/trade_tables_creation.sql

```diff
@@ -0,0 +1,46 @@
+-- psql -U username -d noshirt -a -f trade_tables_creation.sql
+
+CREATE TABLE IF NOT EXISTS pair_status (
+    status_id int primary key not null,
+    status varchar(25)
+);
+
+CREATE TABLE IF NOT EXISTS pair (
+    pair_id serial primary key,
+    pair_code varchar(25) not null,
+    active boolean not null default false,
+    status_id int REFERENCES pair_status (status_id)
+);
+
+CREATE TABLE IF NOT EXISTS exchange_config (
+    id varchar(120) NOT NULL,
+    sk varchar(120) NOT NULL,
+    exchange varchar(25) NOT NULL
+)
+
+CREATE TABLE IF NOT EXISTS initial_config (
+    exchange varchar(25) NOT NULL,
+    pair varchar(30) NOT NULL,
+    intraday_interval varchar(5) NOT NULL,
+    trend_interval varchar(5) NOT NULL,
+    trend_class varchar(60) NOT NULL,
+    strategy_class varchar(3) NOT NULL,
+    max_open_orders integer,
+    order_value real NOT NULL,
+    max_risk real
+);
+
+
+DELETE FROM public.pair_status;
+
+INSERT INTO public.pair_status(status_id, status)
```

Comment:
Não entendi 100% o conceito dessa tabela. Por exemplo, me parece q essa opção`available` tem como finalidade controlar se a moeda está disponível ou não para entrar em novas operações. É isso?
Se for, acho que é uma configuração desnecessária... A gente já consegue obter essa informação da tabela `Trade`. Se o pair não está em nenhuma operação aberta, logo o pair está disponível para novas entradas... 
Essa "redundancia" de informações em mais de uma tabela as vezes mais dificulta do que facilita!